packages/experiments/src/components/ExtendedPicker/BaseExtendedPicker.tsx

```diff
@@ -0,0 +1,100 @@
+/* tslint:disable */
+import * as React from 'react';
+/* tslint:enable */
+import { BasePicker, BaseAutoFill } from 'office-ui-fabric-react/lib/Pickers';
+import { autobind, KeyCodes } from '../../Utilities';
+import { IPersonaProps } from 'office-ui-fabric-react/lib/Persona';
+import { IObjectWithKey } from 'office-ui-fabric-react/src/utilities/selection/index';
+import { IBaseFloatingPickerProps } from './BaseFloatingPicker.Props';
+import { IBaseExtendedPickerProps } from './BaseExtendedPicker.Props';
+import { BaseFloatingPicker } from './BaseFloatingPicker';
+
+export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>> extends BasePicker<T, P> {
+
+  protected floatingPicker: BaseFloatingPicker<IPersonaProps, IBaseFloatingPickerProps<IPersonaProps>>;
+
+  public componentDidMount(): void {
+    this.selection.setItems(this.state.items as IObjectWithKey[]);
+    this.forceUpdate();
+  }
+
+  protected renderSuggestions(): JSX.Element | null {
+    let TypedFloatingPicker = this.props.floatingPickerType as (new (props: IBaseFloatingPickerProps<IPersonaProps>) =>
```

Comment:
That's a great suggestion. Since I'm going to be creating 2 new components, selected items list and another that uses that and the floating picker a in a separate PR, I'll implement this suggestion there.